### PR TITLE
upgrade Twitter gem - fixes Faraday deprecation warning

### PR DESCRIPTION
**Pull request by mrbrdo**
tested connect twitter, post on twitter and delete on twitter - working

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -807,7 +824,7 @@ DEPENDENCIES
   spring-commands-rspec (= 1.0.4)
   test_after_commit (= 0.4.1)
   timecop (= 0.7.3)
-  twitter (= 4.8.1)
+  twitter (= 5.14.0)
   twitter-text (= 1.11.0)
   typhoeus (= 0.7.1)
   uglifier (= 2.7.1)
```
Comment (jhass) [inline Gemfile.lock]: Please commit `Gemfile.lock` changes after running `bin/spring stop; DB=mysql bundle` for now.

**Comment by jhass** [comment]
Hey, looks like a few specs need updates: https://travis-ci.org/diaspora/diaspora/jobs/60016505#L1690

https://wiki.diasporafoundation.org/Testing_workflow has some hints on how to get them running locally.

File: Gemfile.lock [hunk 2]
```diff
@@ -152,8 +153,11 @@ GEM
       devise (~> 3.4.0)
     devise_lastseenable (0.0.4)
       devise
+      devise
+      rails (>= 3.0.4)
       rails (>= 3.0.4)
       warden
+      warden
```
Comment (SuperTux88) [inline Gemfile.lock]: These entries are now doubled.

Comment (mrbrdo) [inline Gemfile.lock]: I noticed, but no idea why, just did bundle update.

**Comment by denschub** [comment]
Superseded by #6083.

**Pull request closed without merging**